cmd/red-box/main.go

```diff
@@ -81,3 +75,21 @@ func main() {
 	}
 	wg.Wait()
 }
+
+func config(path string) (sgrpc.Config, error) {
+	if path == "" {
+		// return default value for map so any read will return
+		// default value for PartitionResources
+		return nil, nil
```

Comment:
can't we return an empty config instead nil?

Reply:
We can, but that will the same. 
Reusing zero values is more go-way IMO.